tests/common/dualtor/dual_tor_utils.py

```diff
@@ -872,6 +872,7 @@ def _generate_hashed_ipv4_packet(src_mac, dst_mac, dst_ip, hash_key):
         exp_tunnel_pkt.set_do_not_care_scapy(IP, "id")       # since src and dst changed, ID would change too
         exp_tunnel_pkt.set_do_not_care_scapy(IP, "ttl")      # ttl in outer packet is set to 255
         exp_tunnel_pkt.set_do_not_care_scapy(IP, "chksum")   # checksum would differ as the IP header is not the same
+	exp_tunnel_pkt.set_do_not_care_scapy(IP, "flags")  # "Don't fragment" flag may be set in the outer header
```

Comment:
not sure if this is the right way or not but all the comments are aligned and the new added one is not. could you please align the indentation of the comment as well?

Reply:
Hi @liat-grozovik，I've aligned the comments and fixed the pre-commit failure. Thanks for your comments!